src/main/java/seedu/address/model/attribute/Email.java

```diff
@@ -9,23 +9,26 @@
  */
 public class Email {
 
-    private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
+    private static final String SPECIAL_CHARACTERS = "\\._%+-";
```

Comment:
Would suggest to update the UserGuide as well regarding the type of special characters allowed. The user guide has a section explaining the valid format of emails.

Reply:
Done